### DISCUSSION TITLE
Link in github release notes to CHANGELOG.md

### DISCUSSION THREAD

**Discussion by meliache** [latex-lsp/texlab#847] [Ideas]
I like that you have a `CHANGELOG.md` but it would also be useful to have a changelog in the github [releases](https://github.com/latex-lsp/texlab/releases) page or at least link to the CHANGELOG.md from the release page. I usually subscribe to the Atom feed of the releases via  [releases.atom](https://github.com/latex-lsp/texlab/releases.atom), other people might get email notifications via the `Watch` function in github.

Otherwise I'm not sure if it's possible to subscribe to a  `CHANGELOG.md`. Through googling I found that I can subscribe to a feed of commits to that file, but that's not exactly the same.

**Comment by pfoerster** [comment]
@meliache Sure, we could use something like https://github.com/anton-yurchenko/git-release, which automatically includes the changelog in the release based on the `CHANGELOG.md` file.
